### PR TITLE
[typo fix] fix some typo errors 

### PR DESCRIPTION
**Pull request by gentleking**
typo errors in different dirs including doc/comment typo error and code typo error

### CODE REVIEW

File: demos/local_attestation/DiffieHellmanLibrary/Makefile [hunk 1]
```diff
@@ -8,7 +8,7 @@ CC = occlum-gcc
 CXX = occlum-g++
 CFLAGS =  -I$(OPENSSL)/include -L$(OPENSSL) -lcrypto -fPIC
 SOFLAGS = -shared  $(CFLAGS)
-CFLAGS += -I$(SGX_SDK)/include -Iinclude
+CFLAGS += -I$(SGX_SDK)/include - include
```
Comment (volcano0dr) [inline demos/local_attestation/DiffieHellmanLibrary/Makefile:11]: this is not a typo.

**Comment by volcano0dr** [comment]
@gentleking Can you squash all commits?

File: demos/local_attestation/DiffieHellmanLibrary/Makefile (hunk 1 above)
Comment (gentleking) [reply demos/local_attestation/DiffieHellmanLibrary/Makefile]: ok, I have fixed this error.

**Comment by gentleking** [comment]
ok, I have squashed all commit. Please review again.

File: demos/local_attestation/DiffieHellmanLibrary/Makefile (hunk 1 above)
Comment (BIIIANG) [reply demos/local_attestation/DiffieHellmanLibrary/Makefile]: > ok, I have fixed this error.

I think there is still an extra space in your latest commit.
[Signed-off-by: 景涛 <jingtao3@hust.edu.cn>](https://github.com/occlum/occlum/pull/1237/commits/ddf2d58e3f99e414507c60a12cc96b0e18432ad9)

Comment (gentleking) [reply demos/local_attestation/DiffieHellmanLibrary/Makefile]: thanks, I have fixed this error

**Review by volcano0dr — APPROVED** [review]
LGTM

**Pull request merged**